CHANGELOG

```diff
@@ -1,3 +1,7 @@
+1.2.0 (2020-04-24)
+---------------------------------------
+Bump Scala Common Enrich to 1.1.0 (#29)
+
 1.1.0 (2020-02-??)
```

Comment:
Can you please create a ticket to fix this, @oguzhanunlu.

Reply:
We could take the opportunity to fix all the dashes of the file at the same time.